README_RASPBERRY_PI.md

```diff
@@ -0,0 +1,192 @@
+* Optional: USB soundcard with headphones or speakers connected
+
+Building
+--------
+
+### Step 1: Hardware setup
+
+1. Connect an ethernet cable from the network router to the RPi
+2. Insert the SD card and USB soundcard. For a GUI build, connect screen, mouse and keyboard
+3. Connect USB power from a power supply
+
+### Step 2: Update the system, install required libraries
+
+For a GUI-less build, first ssh into the RPi from your laptop by opening a terminal and typing:
+
+    ssh pi@raspberrypi # default password is 'raspberry'
+
+In a terminal, type (or copy-and-paste):
+
+    sudo raspi-config # change password and optionally edit hostname and enable ssh and/or vnc
+    sudo apt-get update
+    sudo apt-get upgrade
+    sudo apt-get dist-upgrade
+
+Install required libraries:
+
+    # For GUI builds:
+    sudo apt-get install libjack-jackd2-dev libsndfile1-dev libasound2-dev libavahi-client-dev \
+        libreadline-dev libfftw3-dev libxt-dev libudev-dev cmake qttools5-dev qttools5-dev-tools \
+        qtdeclarative5-dev libqt5svg5-dev qjackctl
+    # For GUI-less builds:
+    sudo apt-get install libsamplerate0-dev libsndfile1-dev libasound2-dev libavahi-client-dev \
+        libreadline-dev libfftw3-dev libudev-dev cmake git
+
+### Step 3: GUI-less builds only: compile and install jackd (no d-bus)
+
+    cd ~ # or cd into the directory where you'd like to build jack2
+    git clone git://github.com/jackaudio/jack2 --depth 1
+    cd jack2
+    ./waf configure --alsa --libdir=/usr/lib/arm-linux-gnueabihf/
+    ./waf build
+    sudo ./waf install
+    sudo ldconfig
+    cd ..
+    rm -rf jack2
+    sudo sh -c "echo @audio - memlock 256000 >> /etc/security/limits.conf"
+    sudo sh -c "echo @audio - rtprio 75 >> /etc/security/limits.conf"
+    exit # and ssh in again to make the limits.conf settings work
+
+### Step 4: Compile and install SuperCollider
+
+    cd ~ # or cd into the directory where you'd like to clone and build supercollider
+    git clone --recurse-submodules https://github.com/supercollider/supercollider.git
+    cd supercollider
+    mkdir build && cd build
+
+    # For a GUI-less build:
+    cmake -DCMAKE_BUILD_TYPE=Release -DSUPERNOVA=OFF -DSC_ED=OFF -DSC_EL=OFF -DSC_VIM=ON -DNATIVE=ON \
+        -DSC_IDE=OFF -DNO_X11=ON -DSC_QT=OFF ..
+
+    # For a GUI build:
+    cmake -DCMAKE_BUILD_TYPE=Release -DSUPERNOVA=OFF -DSC_ED=OFF -DSC_EL=OFF -DSC_VIM=ON -DNATIVE=ON \
+        -DSC_USE_QTWEBENGINE:BOOL=OFF ..
+
+    cmake --build . --config Release --target all -- -j3 # use -j3 flag only on RPi3 or newer
+    sudo cmake --build . --config Release --target install
+    sudo ldconfig
+
+### Step 5: Set up JACK
+
+    # `-dhw:0` is the internal soundcard. Use `-dhw:1` for USB soundcards. `aplay -l` will list available devices.
+    # Use `nano ~/.jackdrc` to edit jack settings.
+
+    # For GUI builds:
+    echo /usr/bin/jackd -P75 -dalsa -dhw:0 -r44100 -p1024 -n3 > ~/.jackdrc
+
```

Comment:
suggestion: add `-p16` to the above line so that it's the same all over.  something i missed in the original.